modules/miscutil/lib/orcidutils.py

```diff
@@ -593,9 +581,10 @@ def _description_weight(description):
         short_descriptions = \
             record_get_field_values(recstruct, '520', '', '', 'a')
         if short_descriptions:
-            work_dict['short_description'] = max([encode_for_jinja_and_xml(sd)
-                                                 for sd in short_descriptions],
-                                                 key=_description_weight)
+            work_dict['short_description'] = encode_for_jinja_and_xml(
+                                             short_descriptions[0])[
+                                             :MAX_DESCRIPTION_LENGTH
+                                             ].rsplit(' ', 1)[0]
```

Comment:
I guess it will crash as usual in case of an empty MARC field... but at that point... too bad.